pkg/interface/src/views/components/s3-upload.tsx

```diff
@@ -1,16 +1,30 @@
 import React, { Component } from 'react'
-import { Icon } from "@tlon/indigo-react";
+import { Box, Text, Icon } from "@tlon/indigo-react";
 
 import S3Client from '~/logic/lib/s3';
 import { Spinner } from './Spinner';
 import { S3Credentials, S3Configuration } from '~/types';
 import { dateToDa, deSig } from '~/logic/lib/util';
 
 export const SubmitDragger = () => (
-  <div
-    className="top-0 bottom-0 left-0 right-0 absolute bg-gray5 h-100 w-100 flex items-center justify-center z-999"
-    style={{pointerEvents: 'none'}}
-  >Drop a file to upload</div>
+  <Box
+    top='0'
+    bottom='0'
+    left='0'
+    right='0'
+    position='absolute'
+    backgroundColor='white'
+    height='100%'
+    width='100%'
+    display='flex'
+    alignItems='center'
```

Comment:
It's probably cleaner to write this a `<Center>` because it automatically sets the height, flex, and alignment props to the one you want.

Reply:
It apparently does not support `backgroundColor` in 1.1.5 so we don't get the colour scheme stuff here.

Reply:
Ah don't worry about it then!